src/data/normalizer.ts

```diff
@@ -120,6 +121,59 @@ export const normalizeDataForDataConfig = async (
     return outputData;
 };
 
+const normalizeNtVariants = async (
+    variants: Entry<any>[],
+    service: ContentfulNormalizerService
+) => {
+    return await Promise.all(
+        variants.map(async (variant: any): Promise<Entry<any>> => {
+            return await normalizeData(variant, service);
+        })
+    );
+};
+
+const addNinetailed = async (
+    outputData: any,
+    data: Entry<any>,
+    service: ContentfulNormalizerService
+): Promise<void> => {
+    if (!data || !data.fields || !data.fields.nt_experiences) {
+        return;
+    }
+
+    const ntFieldData = getValueOfField(data.fields.nt_experiences, service.locale);
+
+    if (!ntFieldData) {
+        return;
+    }
+
+    let variants: Entry<any>[] = [];
+    const rawVariants: Record<string, Entry<any>> = {};
+
+    ntFieldData.forEach((data: Entry<any>) => {
+        variants = data.fields.nt_variants;
```

Comment:
Is it intended to overwrite the entire variable on every iteration? You do not want to append?

Reply:
no we don't want to append here, it's intended. But what i see is that we got an circular reference error if there are more than 2 variants.